site/content/tutorial/04-logic/05-keyed-each-blocks/text.md

```diff
@@ -2,18 +2,20 @@
 title: Keyed each blocks
 ---
 
-By default, when you modify the value of an `each` block, it will add and remove items at the *end* of the block, and update any values that have changed. That might not be what you want.
+By default, when you add / remove an item from the iterable that the `each` block iterates over, it will add / remove the component instance at the *end* of the block instead of the instance corresponding to that element of the iterable. If you add / remove an item from anywhere other than the end of the iterable, this leads to each component instance corresponding to a different element of the iterable than the one it was first created for. That might not be what you want.
```

Comment:
`iterable` -> `array`

- not all iterable can be iterated through `{#each}` block, eg: Set
- this is a tutorial, i think we should use more beginner friendly word

Reply:
True, I'll change that.